### ISSUE TITLE
Problem with styles.js

### ISSUE THREAD

**Issue by tyssen** [ryancramerdesign/InputfieldCKEditor#15]
If I add a custom styles.js and point to it from my field's settings, the styles dropdown gets greyed out. Pretty sure I've got everything set up correctly with path to the file and the name in the settings matching the name in the file itself.

So I thought I'd try editing CKEditor's own styles.js but when I do, nothing changes in the CP.

This is with the latest versions of the module and Processwire.

**Comment by teppokoivula** [comment]
@tyssen: I've never set custom path for styles.js, so can't really speak for that, but what I have done is customizing CKEditors own styles.js and for me it's been working properly.

Are you sure that you're not defining custom styles.js _and_ modifying default one simultaneously (when those changes to default file don't seem to change anything?) Is the file getting refreshed properly too, i.e. it's not coming from cache?

**Comment by tyssen** [comment]
Yeah, I removed the path to styles.js when it was coming up greyed out and that brought back the default one. Seems styles.js is aggressively cached or something because I had to use an incognito window to see the changes I was making. Usually a hard refresh sorts out any updates to CSS or js. :?

**Comment by teppokoivula** [comment]
If you're using Chrome, it's pretty damn awesome at never clearing cache. When using dev-tools, right-click > clear browser cache is a lifesaver.
